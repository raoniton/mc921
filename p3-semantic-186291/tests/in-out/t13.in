class Main {
    char a = 'a';
    int a = 97;
    public static void main(String[] args) {
        a = a + 'b';       
    }
}