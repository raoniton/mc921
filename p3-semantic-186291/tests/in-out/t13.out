SemanticError: Field a has already been defined @ 3:9
